### PR TITLE
REF: Rename ColinearityWarning

### LINKED ISSUE #4671: REF: warnings rename ColinearityWarning to CollinearityWarning
https://groups.google.com/d/msg/pystatsmodels/S9VspZLn7No/tBN3AMY7BwAJ

for consistent spelling we should change it in tools.sm_exceptions

### PR DESCRIPTION
**Pull request by bashtage**
Fix typo in warning name

closes #4671

### CODE REVIEW

File: statsmodels/tsa/tests/test_stattools.py [hunk 1]
```diff
@@ -11,12 +11,11 @@
 from statsmodels.compat.numpy import recarray_select
 from statsmodels.compat.python import lrange
 from statsmodels.datasets import macrodata, sunspots
-from statsmodels.tools.sm_exceptions import ColinearityWarning
-from statsmodels.tools.sm_exceptions import MissingDataError
+from statsmodels.tools.sm_exceptions import CollinearityWarning, MissingDataError
```
Comment (josef-pkt) [inline statsmodels/tsa/tests/test_stattools.py]: this line looks too long now, use parenthesis

**Comment by josef-pkt** [comment]
LGTM, minor pep-8

**Comment by josef-pkt** [comment]
It's not backwards compatible, but I doubt anyone checks this warning in code

File: statsmodels/tsa/tests/test_stattools.py (hunk 1 above)
Comment (bashtage) [reply statsmodels/tsa/tests/test_stattools.py]: Lots of places seem to use 100, which is within the pep spec.  I definitely prefer 100.

**Comment by josef-pkt** [comment]
as green as it gets, merging

@bashtage Thank you

**Pull request merged**